### PR TITLE
[W4.11e][F10-4] Neo Kai Jun

### PR DESCRIPTION
**Pull request by HyperionNKJ**
Added JUnit test for isAnyNull() method.

### CODE REVIEW

**Comment by HyperionNKJ** [comment]
Ready for review

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -4,12 +4,38 @@
 import static org.junit.Assert.assertTrue;
 
 import java.util.Arrays;
-import java.util.List;
```
Comment (dalessr) [inline test/java/seedu/addressbook/common/UtilsTest.java:7]: This is considered as an unrelated change since it has nothing to do with your current PR.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -4,12 +4,38 @@
...
+        // has multiple objects
+        assertHasNull(null, "2");
+        assertHasNull(null, null);
+        assertHasNull( 1, null, Integer.valueOf(3));
+        assertHasNoNull("", "abc", "a", "abc");
+        assertHasNoNull(1, Integer.valueOf(1));
+        assertHasNoNull("23", "", " ", "null");
+    }
+
+    private void assertHasNull(Object... objects) {
```
Comment (dalessr) [inline test/java/seedu/addressbook/common/UtilsTest.java:31]: Extracting the assert function is not really necessary here since this method is quite simple (with only one line). Thus, you can add it directly into your test cases.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 3]
```diff
@@ -4,12 +4,38 @@
...
+    public void isAnyNull() {
+        // empty list
+        assertHasNoNull();
+
+        // only 1 object
+        assertHasNull((Object) null);
+        assertHasNoNull(1);
+        assertHasNoNull("");
+
+        // has multiple objects
```
Comment (dalessr) [inline test/java/seedu/addressbook/common/UtilsTest.java:22]: To make the test cases clearer, maybe you can try further separating the case into `// has multiple objects with null` & `// has multiple objects without null`

**Review by dalessr — COMMENTED** [review]
Good attempt in general 👍 @HyperionNKJ . Please see the comments below.

Also, please close this PR after reading the comments.

File: test/java/seedu/addressbook/common/UtilsTest.java (hunk 3 above)
Comment (HyperionNKJ) [reply test/java/seedu/addressbook/common/UtilsTest.java:22]: Okay. Thank you!! :)
